style.css

```diff
@@ -0,0 +1,104 @@
+html,
+body {
+    margin: 0;
+    height: 100%;
+    background-image: url("images/tile.jpg");
+}
+
+@keyframes fadeIn {
+  from { opacity: 0; }
+  to { opacity: 1; }
+  
+}
+
+label{
+  background-color: white;
+}
+
+#container {
+    display: grid;
+    height: 100%;
+    grid-template-rows: 1fr 4fr 8fr;
+     grid-template-columns: 4fr 5fr 4fr;
+}
+
+#header {
+    grid-row: 1;
+    grid-column: 1/ span 3;
+    flex-direction: row;
+    text-align: center;
+}
+
+#form {
+    grid-row: 2;
+    grid-column: 2;
+    background-image: url("images/formbg.jpg");
+    display: flex;
+    flex-direction: column;
+    background-repeat: repeat-y;
+   
+
+}
+
+
+
+h1 {
+    font-family: "Fascinate", system-ui;
+
+}
+
+#tasks-list {
+    display: flex;
+     flex-wrap: wrap;
+
+    overflow :auto;
+    grid-column: 1/ span 3;
+    grid-row: 3;
+    
+}
+
+.task {
+    background-image: url("images/notebg.png");
+
+    background-repeat: no-repeat;
+    width: 180px;
+    height: 300px;
+    display:block;
+    margin-left: 20px;
+    margin-right: 20px;
+   overflow-y: auto;
+  overflow-x: hidden;
+  word-wrap: break-word;
+  font-family:'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif
+     /* margin-left: 20px; 
+     margin-right: 20px */
+     
+}
+
+.fade-in{
+    opacity: 0;
+    animation:  fadeIn 2s ease forwards;
+} 
+
+
+
+
+.delete-btn-div {
+   
+  /* position: relative; */
+  padding: 30px;
+  width: 20px;
+  height: 20px;
+  margin-left: auto;
+  
+  
+}
+
+.delete-btn {
+  display: none;
+  margin-top: 10px;
+}
+
+.delete-btn-div:hover .delete-btn {
+  display: block; 
+}
```

Comment:
in mobile, never waster width
<img width="625" height="725" alt="Screenshot 2025-08-03 at 21 51 59" src="https://github.com/user-attachments/assets/eb0a6ea2-6864-4c32-8c6a-d3433b9d399d" />